### ISSUE TITLE
Could not open input file

### ISSUE THREAD

**Issue by niklasschroetler** [bloodhunterd/Froxlor#5]
When starting the container, I get the following logs:
```
Starting periodic command scheduler: cron.
Starting domain name service...: bind9.
Could not open input file: /var/www/froxlor/scripts/froxlor_master_cronjob.php
```

What do I need to do to fix this?

Thanks in advance!

**Comment by bloodhunterd** [comment]
This should be an easy one.

You need to download [Froxlor](https://froxlor.org/) from the official website first. Then unzip it and mount it into the container. See the [Volumes](https://github.com/bloodhunterd/Froxlor#Volumes) section in the readme file.

Due Froxlor's incompatibility to the Docker concept it isn't build in, but maybe  I should add some more documentation in the future.

**Comment by niklasschroetler** [comment]
It's listed under deployment, you're correct. Maybe add this to the volume description explicitly?
Thanks for the quick reply!

**Comment by CoolNamesAllTaken** [comment]
It looks like this bug has popped up again since Froxlor recently released its [version 2.0](https://github.com/Froxlor/Froxlor/releases/tag/2.0.13), which removed the scripts directory. Any guidance on how to fix this would be much appreciated!
